tests/unit_node/async_hooks_test.ts

```diff
@@ -92,7 +92,7 @@ Deno.test(async function enterWith() {
   });
 
   assertEquals(await deferred.promise, { x: 2 });
-  assertEquals(await deferred1.promise, { x: 1 });
```

Comment:
node.js agrees that this should be null.

Reply:
Probably got out of sync with Node.js tests since this test was added 👍